src/PerfView/GuiUtilities/WindowBase.cs

```diff
@@ -15,8 +15,17 @@ public class WindowBase : Window
 
         public WindowBase(Window parentWindow)
         {
-            Owner = parentWindow;
-            WindowStartupLocation = WindowStartupLocation.CenterOwner;
+            if (parentWindow != null)
+            {
+                // Try to set the owner, but it might fail (if the window has never been displayed)
+                // give up setting the owner in that case (it is not critical to have an owner)  
+                try
+                {
+                    Owner = parentWindow;
+                    WindowStartupLocation = WindowStartupLocation.CenterOwner;
+                }
+                catch (System.Exception) { }
```

Comment:
❓ Can we be more specific about the exception type?

Reply:
The exception that is thrown in the case that caused the issue is an InvalidOperationException.  However the logic goes for any error.    I do understand that assuming that other exceptions don't corrupt state is a risk, but most of that risk is mitigated by the runtime treating 'corruption state exceptions' as uncatchable.   There is risk either way, and I guess I expecting that the likelihood of a truly corrupting state when all you are doing is setting the owner is unlikely (but the possibility of other exceptions (like null reference exceptions) being higher likelihood.